test/seeds/offchainSeeds/offchain.match2.seed.ts

```diff
@@ -48,7 +48,7 @@ export const seededMatch2NotInRoles = '619fb8ca-4dd9-4843-8c7a-9d9c9474784e'
 
 const seededDemandANotOwnedId = 'c88908aa-a2a6-48df-a698-572aa30159c0'
 const seededDemandBNotOwnedId = 'b21f865e-f4e9-4ae2-8944-de691e9eb4d9'
-const seededDemandBWithTokenId = 'b005f4a1-400e-410e-aa72-8e97385f63e6'
+export const seededDemandBWithTokenId = 'b005f4a1-400e-410e-aa72-8e97385f63e6'
```

Comment:
can you move this up to the other exports please

Reply:
yes